src/base/base.pri

```diff
@@ -107,10 +107,10 @@ SOURCES += \
     $$PWD/rss/private/rssparser.cpp \
     $$PWD/utils/fs.cpp \
     $$PWD/utils/gzip.cpp \
+    $$PWD/utils/net.cpp \
```

Comment:
Fixed. BTW, what to do in case of gui.pri, where I have added files too, but alphabet order is already broken?